lib/cypress/cqm_execution_calc.rb

```diff
@@ -12,12 +12,14 @@ def initialize(patients, measures, correlation_id, options)
       @measures = measures
       @correlation_id = correlation_id
       @options = options
+      @ir_list = []
     end
 
     def execute(save = true)
       results = @measures.map do |measure|
         request_for(measure, save)
       end.flatten
+      IndividualResult.create(@ir_list) if save
```

Comment:
Do we want to use *Model.create!* (with an exclamation point), which "Insert a document or multiple documents into the database, raising an error if a validation error occurs." The difference is that it would raise validation errors.